tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```
@@ -676,6 +676,7 @@
   "DD_TRACE_DATA_PIPELINE_ENABLED": "trace_data_pipeline_enabled",
   "DD_TRACE_INJECT_CONTEXT_INTO_STORED_PROCEDURES_ENABLED": "trace_inject_context_into_stored_procedures_enabled",
   "DD_TRACE_GRAPHQL_ERROR_EXTENSIONS": "trace_graphql_error_extensions",
+  "DD_TRACE_BAGGAGE_TAG_KEYS": "trace_baggage_tag_keys"
```

Looks like you're missing a comma, which is causing the corresponding unit test to fail

```suggestion
  "DD_TRACE_BAGGAGE_TAG_KEYS": "trace_baggage_tag_keys",
```